config/crd/kustomization.yaml

```diff
@@ -18,8 +18,7 @@ resources:
   # Including creation of Service here for now, will later be done automatically
   - bases/serving.kserve.io_predictors.yaml
   - bases/serving.kserve.io_inferenceservices.yaml
-  # ClusterServingRuntime not yet supported by modelmesh-serving
-  #  - bases/serving.kserve.io_clusterservingruntimes.yaml
+  - bases/serving.kserve.io_clusterservingruntimes.yaml
```

Comment:
I'm wondering whether we should include this in namespace scope case.

Reply:
Good question. Since we won't do anything with CSRs in namespace scope mode, probably can exclude it during install.

Reply:
@chinhuang007 WDYT about excluding this for namespace mode? We could just have a separate cluster_crd kustomization which includes this one + the CSR CRD